ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -111,6 +111,10 @@ NGraphEncapsulateOp::NGraphEncapsulateOp(OpKernelConstruction* ctx)
   int graph_id{-1};
   OP_REQUIRES_OK(ctx, ctx->GetAttr("ngraph_graph_id", &graph_id));
   ng_encap_impl.SetGraphId(graph_id);
+
+  OP_REQUIRES_OK(ctx, ctx->GetAttr(("_session_name" + to_string(graph_id)),
```

Comment:
Same thing guard the function calls, on whether TB diagnostics is requested.

Reply:
see comment above